src/components/calcite-date-month/calcite-date-month.tsx

```diff
@@ -301,4 +274,59 @@ export class CalciteDateMonth {
     }
     return days;
   }
+
+  /**
+   * Determine if the date is in between the start and end dates
+   */
+  private betweenSelectedRange(date: Date): boolean {
+    return this.startDate && this.endDate && date > this.startDate && date < this.endDate;
+  }
+
+  /**
+   * Determine if the date should be in selected state
+   */
+  private isSelected(date: Date): boolean {
+    return sameDate(date, this.selectedDate) || (this.startDate && sameDate(date, this.startDate));
+  }
+
+  /**
+   * Render calcite-date-day
+   */
+  private renderDateDay(
+    active: boolean,
+    day: number,
+    dir: string,
+    date: Date,
+    currentMonth?: boolean,
+    ref?: boolean
+  ) {
+    const props = {
+      active,
+      ...(currentMonth && { currentMonth: true }),
+      day,
+      dir,
+      disabled: !inRange(date, this.min, this.max),
+      startOfRange:
```

Comment:
it may read a bit better to just make this conditional a method on this component like `this.isStartOfRange`.